themis/_index.md

```diff
@@ -11,25 +11,67 @@ Menu: "main"
 _What Themis is and why it's awesome_
 
 
-## What is Themis
+## What Themis is
+
+Themis is a cross-platform high-level cryptographic library for mobile, web and server platforms. Themis solves 90% of typical data protection use cases that are common for most apps. 
+
+Themis provides ready-made building blocks _(["crypto-systems"](#crypto-systems))_, which simplify usage of core cryptographic security operations.
+
+
+{{< hint info >}}
+Learn more about [cryptography in Themis](/themis/crypto-theory/).
+{{< /hint>}}
+
+
+
+## Why Themis
+
+Unlike many other cryptographic libraries, Themis is a high-level, easy-to-use and hard-to-misuse library _(["boring crypto"](https://speakerdeck.com/vixentael/use-cryptography-dont-learn-it))_. 
+
+Themis is made by cryptographers, but targeted on developers, so it hides cryptographic details under the hood _("secure by design")_.
+
+Themis helps to build both simple and complex cryptographic features easily, quickly and securely. Themis allows developers to focus on the main thing: developing their applications.
+
+{{< hint info >}}
+Learn more about [security design ideas behind Themis](/themis/architecture/) and [how team maintains Themis](https://speakerdeck.com/vixentael/maintaining-cryptographic-library-for-12-languages) for last 6 years.
+{{< /hint>}}
 
-TODO:
 
 
 ## Use cases that Themis solves
 
-TODO:
+* **Encrypt stored secrets** in your apps and backend (API keys, session tokens, files).
```

Comment:
```suggestion
* **Encrypt stored secrets** in your apps and backend: API keys, session tokens, files.
```

I'd use colons instead of parenthesised expressions in most cases here as they expand the use cases, not merely augment or rephrase them.